src/test/java/com/tabnine/unitTests/BinaryProcessRequesterProviderTests.java

```diff
@@ -0,0 +1,173 @@
+package com.tabnine.unitTests;
```

Comment:
y like dissss y not in kotlin :(
please try to convert to kotlin, if its too complicated leave it as is